test/built-ins/Symbol/desc-to-string-symbol.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-symbol-constructor
+es6id: 19.4.1
+description: The first argument is coerced to a String value
```

Comment:
This could use a distinct description from [`test/built-ins/Symbol/desc-to-string.js`](test/built-ins/Symbol/desc-to-string.js
)